Applications/Spire/Source/Login/LoginWindow.cpp

```diff
@@ -52,6 +52,29 @@ namespace {
     return style;
   }
 
+  auto SIGN_IN_BUTTON_STYLE() {
+    auto style = StyleSheet();
+    auto font = QFont("Roboto");
+    font.setWeight(QFont::Bold);
+    font.setPixelSize(scale_width(14));
+    style.get(Any() > Button::Body()).
+      set(TextAlign(Qt::Alignment(Qt::AlignCenter))).
+      set(text_style(font, QColor::fromRgb(0, 0, 0))).
+      set(BackgroundColor(QColor(0x68, 0x4B, 0xC7))).
+      set(border(scale_width(1), QColor(0x68, 0x4B, 0xC7))).
+      set(TextColor(QColor(0xE2, 0xE0, 0xFF)));
+    style.get(Hover() > Button::Body()).
+      set(BackgroundColor(QColor(0x8D, 0x78, 0xEC))).
+      set(border(scale_width(1), QColor(0x8D, 0x78, 0xEC)));
+    style.get(Focus() > Button::Body()).
+      set(border(scale_width(1), QColor(0x8D, 0x78, 0xEC)));
+    style.get(Disabled() > Button::Body()).
+      set(BackgroundColor(QColor(0x4B, 0x23, 0xA0))).
+      set(TextColor(QColor(0x8D, 0x78, 0xEC))).
+      set(border(scale_width(1), QColor(0x68, 0x4B, 0xC7)));
```

Comment:
It's unnecessary to set border size for hover, focus and disabled states. Refer to  Spire::make_label_button